meetballs/settings.py

```diff
@@ -38,7 +38,10 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'meet_balls_app.apps.MeetBallsAppConfig',
-    'player.apps.PlayerConfig'
+    'player.apps.PlayerConfig',
+    'message.apps.MessageConfig',
+    'game_event.apps.GameEventConfig',
+    'player.apps.PlayerConfig',
```

Comment:
notice the duplicate lines, probably an outcome of a rebase.
 'player.apps.PlayerConfig',
 'message.apps.MessageConfig',

Reply:
amzing! thanks @Danielsio